templates/components/plugin.js

```diff
@@ -1,10 +1,31 @@
 import Vue from 'vue'
 <% const components = options.getComponents() %>
 
+const getWrapperFor = (name) => (options) => {
+  if (!options.functional) return options
+
+  const propKeys = Array.isArray(options.props) ? options.props : Object.keys(options.props || {})
+
+  return {
+    name: `${name}Wrapper`,
```

Comment:
- What it will be named without specify name in devtools?
- If name is important, shall we use something like FunctionalWrapper suffix?

Reply:
As above it would have been named the pascal cased version of the component from the directory/filename + 'Wrapper'. (This guarantees it will match the name used by user in template.) But I can rename to use the name of the component if provided...

Reply:
(my original question was that what happens if not setting any `name`) And if we have to, using `FunctionalWrapper` instead of wrapper to make purpose clear.

Reply:
It will be called the name of the original component.

Reply:
Isn't that better to make workaround invisible? (I'm mostly curious of the initiative behind setting name)

Reply:
Good point. 🙈 Originally I thought better to make explicit as an additional entry in devtools lbut as original component doesn't also appear I think invisible is good 👍